app/grants/views.py

```diff
@@ -35,6 +36,11 @@
 from marketing.models import Keyword
 from web3 import HTTPProvider, Web3
 
+now = datetime.datetime.now()
```

Comment:
Can we move this into the method itself? @captnseagraves

Reply:
sure